### PR TITLE
~Ready~ Simple Spirituality for Survivors

### PR DESCRIPTION
**Pull request by Rivet-the-Zombie**
**New trait:** `Spiritual` positive, 1 point
**New item:** `holy symbol`
**New iuse action:** `meditate` +5 morale, +10 max, requires `Spiritual` to perform
**New item flag:** `INSPIRATIONAL` added to some existing books

Adds a new starting-only one point positive trait, `Spiritual`, which interacts with various ways the character's morale can be manipulated.

Characters possessing the `Spiritual` trait gain a small amount of bonus morale when reading holy books (of any sort) or other books with the `INSPIRATIONAL` flag.

Characters possessing the `Spiritual` trait gain bonus morale when  consuming certain natural entheogens (`mushroom_magic`).

Adds a new item, `holy symbol`, id `holy_symbol`, which uses the new `meditate` iuse function, allowing characters with the `Spiritual` trait to spend 2000 moves (~2 minutes) to gain five points of temporary morale, capped at ten (two uses). This function can be used an infinite number of times, and requires no charges. The extant `relic` item has been given the `meditate` function as well, and as such can now be used for the same purpose. Characters without the `Spiritual` trait cannot use these items to gain their bonus.

Adds some interactions within the cannibalism code, dealing with the philosophical and spiritual implications of eating people. This generally means different messages and slightly different morale bonuses.

**New item,** `holy_symbol` **spawns:**

museum_misc
church
bedroom
dresser
antique
jewelry_front
spider

**_All effects apply to all faiths and philosophies equally. Every possible effort has been taken to ensure that all faiths have been treated respectfully and that no religion or philosophy is advocated or advanced by the messages and effects within this PR.**_

---

_I've kept it all quite vague/equally applied so that the player is free to decide what their character's spirituality actually means, and no path is objectively 'better' than another._

### CODE REVIEW

**Comment by macrosblackd** [comment]
Possible to turn this off with the remove religious crap mod?

**Comment by Shoes01** [comment]
I would imagine that's more work than simply not choosing the spiritual trait. I don't know if you would spawn with the holy symbol though even with the religious items off.

**Comment by Rivet-the-Zombie** [comment]
Nobody starts with any new items, and that mod removes religious books so it wouldn't affect the new necklace.

File: data/json/items/books.json [hunk 1]
```diff
@@ -3096,7 +3097,8 @@
         "fun" : -1,
         "skill" : "none",
         "price" : 1500,
-        "required_level" : 0
```
Comment (KA101) [inline data/json/items/books.json]: We should probably toughen the requirements for reading the priest's diary as Latin isn't exactly common knowledge.  ('Twas a quest item, IIRC.)

**Comment by KA101** [comment]
What, no pentacle?  ;-P

Only real complaint I've got is that the symbols are both specified and unmarked-randomized.  If I'm practicing X religion but every "holy symbol" doesn't match, likely to be annoying.  (Not sure how much of a problem the Linux shirts are for people using the "wrong" distribution, but T-shirts can be processed for rags.)

File: data/json/items/books.json (hunk 1 above)
Comment (Rivet-the-Zombie) [inline data/json/items/books.json]: But what skill would it use? We don't have any sort of 'vague academia' skill.

**Comment by Rivet-the-Zombie** [comment]
I could nix the random description bit, but I figured this was a good item to use it on.

File: data/json/items/books.json (hunk 1 above)
Comment (KA101) [inline data/json/items/books.json]: High IN?  15 or so ought to be enough.

**Comment by KA101** [comment]
Meh.  The tildes were a bit overboard, thanks much.  I'd say either one generi-item, or separate items so folks don't have to check every holy symbol they find.

**Comment by Rivet-the-Zombie** [comment]
~I like tildes. They're curly and neat.~

**Comment by KA101** [comment]
*shrug*  Just another day in DDA.

**Comment by KA101** [comment]
I presume you intentionally left out datura.

**Comment by Rivet-the-Zombie** [comment]
Oh! It completely slipped my mind. I can add that in a separate PR, though.

**Comment by Snaaty** [comment]
I like this PR.

**Comment by Soyweiser** [comment]
Like the mod myself. Sorry for the necro, but perhaps we could add holy_symbol to the no religious books mod? So this gets removed also for people who are against that kind of thing popping up in CDDA.

**Comment by kevingranade** [comment]
That's part of a NPC quest, so until someone feels like extending the
blacklisting support to the extent that missions can be disabled based on
their contents, that's not going to happen.

**Pull request merged**